Features/PlayingNow/PlayingNowCollectionViewController.swift

```diff
@@ -43,8 +43,12 @@ class PlayingNowCollectionViewController: UICollectionViewController {
             switch result {
             case .success(let moviesList):
                 self.movies.append(contentsOf: moviesList)
-            case .failure(_):
-                break
+            case .failure(let error):
+                DispatchQueue.main.async {
+                    let vc = ErrorViewController(error: error)
+                    vc.modalPresentationStyle = .popover
+                    self.present(vc, animated: true)
+                }
```

Comment:
What about wrapping this in a method to be easily called from any place? Maybe:
```swift
class ErrorViewController {
    static func handleError(_ error: MovieServiceError, presentingViewController: UIViewController) { ... }
}
```
Then in any place you want to use we can simply call:
```swift
ErrorViewController.handleError(error, presentingViewController: self)
```

Reply:
Beautiful!

I was thinking about it but didn't work out the way to implement.